src/tsserver/server.ts

```diff
@@ -57,6 +58,18 @@ namespace ts.server {
         console.warn = (...args) => logger.msg(args.length === 1 ? args[0] : args.join(", "), Msg.Err);
         console.error = (...args) => logger.msg(args.length === 1 ? args[0] : args.join(", "), Msg.Err);
 
+        let fshost;
+        if (vfs) {
+            fshost = new VirtualFS.VirtualServerHost({
+                useCaseSensitiveFileNames: sys.useCaseSensitiveFileNames,
+                executingFilePath: "", // TODO: "" is the default..maybe this should be vfs, vfs:// or .

```

Comment:
This should be where libFile is going to be .. so how are you going to manage lib files ? Your tests do not cover that part

Reply:
I thought that vs code would include them in its initial UpdateFileSystem command, at least that's how I have it modelled in the test right now. I'm not sure if that will actually work -- it *kind of* makes sense because vs code will have to install typescript and start the language server, so it will have to know where the lib files are as well.